### ISSUE TITLE
[Automate] failed handling the TimeEvents.TimeOfDayChanged event

### ISSUE THREAD

**Issue by corjba** [Pathoschild/StardewMods#63]
```
[ ERROR SMAPI] A mod failed handling the TimeEvents.TimeOfDayChanged event:
System.NullReferenceException: Object reference not set to an instance of an object.
   at Pathoschild.Stardew.Automate.Framework.TrackedItem.get_Count() in D:\source\_Stardew\Pathoschild.StardewMods\Automate\Framework\TrackedItem.cs:line 33
   at Pathoschild.Stardew.Automate.Framework.Extensions.TryPush(IPipe[] pipes, ITrackedStack item) in D:\source\_Stardew\Pathoschild.StardewMods\Automate\Framework\Extensions.cs:line 116
   at Pathoschild.Stardew.Automate.AutomateMod.ProcessMachines(MachineMetadata[] machines) in D:\source\_Stardew\Pathoschild.StardewMods\Automate\AutomateMod.cs:line 161
   at Pathoschild.Stardew.Automate.AutomateMod.TimeEvents_TimeOfDayChanged(Object sender, EventArgsIntChanged e) in D:\source\_Stardew\Pathoschild.StardewMods\Automate\AutomateMod.cs:line 99
   at StardewModdingAPI.Framework.InternalExtensions.SafelyRaiseGenericEvent[TEventArgs](IMonitor monitor, String name, IEnumerable`1 handlers, Object sender, TEventArgs args) in D:\source\_Stardew\SMAPI\src\StardewModdingAPI\Framework\InternalExtensions.cs:line 71
```

I keep getting the above error. Whenever I load the game, the first time tick seems to at least partially work, but then after that, Automate stops working. Not sure which machine/item is causing it.

**Comment by corjba** [comment]
I think I tracked it down to the loom.

**Comment by Pathoschild** [comment]
Fixed in the upcoming Automate 1.3 release. Thanks for reporting it. :)
